### DISCUSSION TITLE
how to backup rook ceph，can I use velero？

### DISCUSSION THREAD

**Discussion by 13567436138** [rook/rook#13593] [Q&A]
how to backup rook ceph，can I use velero？if backup cluster and restore cluster ‘s  osd number is different ，what will happen。do I need velero for csi plugin？

**Comment by subhamkrai** [comment]
@13567436138 This section discusses about velero backup https://rook.github.io/docs/rook/latest/Troubleshooting/disaster-recovery/?h=veler#backing-up-and-restoring-a-cluster-based-on-pvcs-into-a-new-kubernetes-cluster

**Reply by 13567436138** [reply]
https://rook.github.io/docs/rook/latest/Troubleshooting/disaster-recovery/?h=veler#adopt-an-existing-rook-ceph-cluster-into-a-new-kubernetes-cluster  
is this did not migrate the data

**Reply by 13567436138** [reply]
@subhamkrai what you provided is for pvcs,what is pvcs mean for that?
